ansible/site.yml

```diff
@@ -18,6 +18,19 @@
   roles:
     - logging
     - gateway
+  tasks:
+    - shell: >
+        ip address show dev ens4
+        | grep -oP 'inet (\d+(\.\d+){3})'
+        | awk '{print $2}'
+      register: GCP_NETWORK_INTERFACE_GATEWAY_IP_ADDRESS
```

Comment:
Use `ansible_default_ipv4.address` which is a built-in fact for ip in default route, which is exactly what we want here. As a bonus we also don't have to hard-code the interface name.

Reply:
Thanks! Unaware of this! =)